apisix/schema_def.lua

```diff
@@ -729,7 +729,18 @@ _M.credential = {
     type = "object",
     properties = {
         -- metadata
-        id = id_schema,
+        id = {
+            oneOf = {
+                id_schema,
+                -- id: "${consumer_name}/credentials/${credential_id}" from standalone mode
+                {
+                    type = "string",
+                    minLength = 15,
+                    maxLength = 128,
+                    pattern = [[^[a-zA-Z0-9]+/credentials/[a-zA-Z0-9]+$]],
+                }
+            }
```

Comment:
If this is standalone-specific, it should be overridden when standalone is enabled, rather than using oneOf.
oneOf is ambiguous at some point and is not conducive to various downstream components using its type.

Reply:
Now, instead of modifying the schema after making a deep copy, this module just modifies it, which is as expected?